### ISSUE TITLE
Automatically setup schedule repairs for each cluster

### ISSUE THREAD

**Issue by sebbonnet** [spotify/cassandra-reaper#142]
We've added support for automatically setting scheduled repairs for keyspaces in a cluster - in our local copy. Let me know if this is a feature you'll be interested in, so I can submit the changes.

Features:
- Schedule repairs are automatically created for all keyspaces within a cluster when the cluster is added.
- Cluster keyspaces are periodically checked for changes, so that the scheduled repairs are always in sync with the cluster keyspaces - i.e. scheduled repairs are added for new keyspaces and removed for deleted keyspaces
- Configuration options are available to turn on/off auto scheduling, specify when the scheduling repairs will be activated and how often the keyspaces sync is done.

**Comment by Bj0rnen** [comment]
Yes, that could be a good feature to have. But leave the default to off, because there can definitely be those keyspaces/tables that have no use for repairs (high throughput, all with a small TTL). Might be useful then to be able to opt out specific keyspaces/tables, but one feature at a time :)

**Comment by sebbonnet** [comment]
@Bj0rnen here is the PR for it. Please note that i've added compatibility with java 1.8 - I hope that's ok too
